src/components/Posts/ComposeReply.tsx

```diff
@@ -59,7 +63,7 @@ const ComposeReply = ({ placeholder }: ComposeReplyProps) => {
         "http://localhost:3001/api/posts/postReply",
         {
           userId: user.userId,
-          parentPostId: post.postId,
+          parentPostId: parentPostId,
```

Comment:
Can simplify this a little since the object key is the same as the variable name

```suggestion
          parentPostId,
```